upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,20 @@
 == Migration Changes
 
+=== Migrating to 8.0.0
```

Comment:
Is there potential to affect current deployments and any migration needed? If not, this should be placed rather into release_notes/topics/8_0_0.adoc

Reply:
Done!